starter-code/HeterogeneousRecyclerView/app/src/main/java/com/charlesdrews/heterogeneousrecyclerview/MainActivity.java

```diff
@@ -31,12 +44,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create and add some players
         // Copy top 2 players in fantasy stats at each position from http://fantasy.nfl.com/research/rankings
-        list.add(new Player("Cam", "Newton", "Jaguars", "QB"));
+        list.add(new Player("Cam", "Newton", "Panthers", "QB"));
```

Comment:
I saw a cat logo and Jaguars was the first cat team name that popped into my head ;)